cluster/ci/config/prow/plugins.yaml

```diff
@@ -1240,6 +1240,10 @@ config_updater:
     cluster/ci/monitoring/build_cop.json:
       name: grafana-dashboard-build-cop
       namespace: prow-monitoring
+    core-services/ci-search/_config.yaml:
+      name: job-config
+      namespace: ci-search
+      key: config.yaml
```

Comment:
We do not need to do this separately, the CM already exists and the content is up2date.